src/test/java/org/apache/commons/lang3/StringUtilsTest.java

```diff
@@ -1226,6 +1241,8 @@ public void testJoin_ArrayOfShorts() {
         assertNull(StringUtils.join((short[]) null, SEPARATOR_CHAR, 0, 1));
         assertEquals(StringUtils.EMPTY, StringUtils.join(SHORT_PRIM_LIST, SEPARATOR_CHAR, 0, 0));
         assertEquals(StringUtils.EMPTY, StringUtils.join(SHORT_PRIM_LIST, SEPARATOR_CHAR, 1, 0));
+        assertEquals("1,2", StringUtils.join(SHORT_PRIM_LIST, SEPARATOR));
```

Comment:
@HubertWo 
The point of String vs char separators is that a String can be longer than one character, so you should test for String separators longer than one character to make sure the right work takes place under the covers.

Reply:
Sorry for the late reply, I was on vac. 
I will take a look on your comments and try to apply changes later this week.

Reply:
The joining part is done via ```java.util.StringJoiner``` with accepts only```CharSequence``` in constructor, so it's made to use more than one character as separator by default. Don't you think that testing that would be redundant?